src/main/java/com/mineinabyss/mobzy/pathfinders/Navigation.kt

```diff
@@ -1,27 +1,24 @@
 package com.mineinabyss.mobzy.pathfinders
 
-import net.minecraft.server.v1_15_R1.*
+import net.minecraft.server.v1_15_R1.EntityInsentient
+import net.minecraft.server.v1_15_R1.EntityLiving
+import net.minecraft.server.v1_15_R1.GenericAttributes
+import net.minecraft.server.v1_15_R1.NavigationAbstract
 import org.bukkit.craftbukkit.v1_15_R1.entity.CraftEntity
 import org.bukkit.entity.Entity
 import org.bukkit.entity.LivingEntity
 
 /**
- * Original methods by Yannick Lamprecht under the MIT license from https://github.com/yannicklamprecht/PathfindergoalAPI
+ * Original methods by Yannick Lamprecht under the MIT license from [PathfindergoalAPI](https://github.com/yannicklamprecht/PathfindergoalAPI)
  */
 
 class Navigation(private val navigationAbstract: NavigationAbstract, private val handle: EntityInsentient) {
-
-    val doneNavigating
-        get() = navigationAbstract.n()
-
-    val pathSearchRange
-        get() = handle.getAttributeInstance(GenericAttributes.FOLLOW_RANGE).value.toFloat()
-
+    val doneNavigating get() = navigationAbstract.m()
```

Comment:
you should look at bedrock scripting for thoughts on how to structure this code https://bedrock.dev/